organization/_posts/2021/2021-04-01-coordination.md

```diff
@@ -0,0 +1,171 @@
+    - Eduardo - is it necessary to have a named liaison? Will end up being the same, well known, people.
+    - Michel - it's not the same as an experiment role; it's lightweight and overcomes diffusion of responsibility
+    - WGs should feedback...
+        - *Main aim is to establish a point of contact and not have a dead channel*
+- Next Hackathon will be organized next June, still a few months ahead. To keep momentum, we will send an invitation for a follow-up of the last Hackathon to be held in April. 
+- IRIS-HEP will resubscribe to The Carpentries. We will take advantage using the resources available, like a training of trainers for HSF.
+
+### Event Generators
+
+Had LHCC Review 2 kickoff [meeting](https://indico.cern.ch/event/1019213/)
+
+- Attendance from most generators/tools groups, will follow up with those missing.
+- Starting to plan next meetings with a view to the HL-LHC review: we will invite each generator/tool group to give contributions over the next ~2 months.
+- Also discussed an update about CMS accounting for generator CPU costs.
+
+### Frameworks
+
+- Working to form panel for reviewing DUNE's framework requirements.  Will be sending out invitations to potential panel members soon.
+- "General discussion" meeting next Wednesday. (With short report from ATLAS's / Gaudi's TBB migration.)
+
+## Other Interest and Activity Areas
+
+### Visualisation
+
+Phoenix people want to start holding meetings as their user base grows - created a category for that in Indico.
+
+### Licensing
+
+ATLAS had reason to use an EvtGen header in Athena, for which it turns out the licence is GPLv3, which would have its usual viral impact on the rest of the code, which is baseline Apache2.
+
+We contacted the EvtGen authors who were not in principle averse to changing to a more liberal licence, however they are dependent on Pythia8, Photos++ and Tauola++, all of which are GPL, tying their hands.
+
+In the course of this discussion we realised that HepMC3 is also GPL. As the main interface between the generators and the experiment codes we would like to ask that the authors change this to (at least) LGPL. This is to be followed up.
+
+Archeology showed that HepMC2 should have been under BSD license.
+
+FastJet authors were not very willing to change their license when we discussed with them. There has been some recent development here to add thread-safety.
+
+Propose dual licensing to generator authors? If they want to keep GPL for everyone and allow HEP to use it non-virally nonetheless, maybe this can be formalised?
+
+*Follow-up in a Generator WG meeting.*
+
+---
+
+## AOB
+
+AIDAinnova kick-off meeting, 13-15 April: <https://indico.cern.ch/event/1003419/overview>.
+
+* There is a software work package with tasks in Turnkey Software, ML Simulation, Tracking, Particle Flow. People very welcome to join the discussion and hear about plans.
+* Frank Gaede (DESY) and Graeme Stewart (CERN) are WP coordinators.
+
```

Comment:
The EOSC-Future project containing the Dark Matter Test Science Project (end to end analysis workflows on the European Open Science Cloud, initially for experiments within ESCAPE <https://projectescape.eu>) has officially started on April 1st, and the next 6 months will be the "planning and preparation" stage prior to starting the work. Caterina Doglioni is working on that and will report back at the next coordination meeting, as it would be nice to create links / be helpful to the activities of the Data Analysis WG.

Reply:
Thanks @urania277 - let's table that for discussion at the next coordination meeting!